spark/src/test/scala/org/apache/mahout/sparkbindings/io/IOSuite.scala

```diff
@@ -187,7 +187,7 @@ object IOSuite {
     val input = new Input(new ByteArrayInputStream(baos.toByteArray))
 
     def outStream: Stream[T] =
-      if (input.eof) Stream.empty
+      if (input.available() == 0) Stream.empty
```

Comment:
You sure? I never quite got those kryo streams. in regular streams, available != eof